src/components/react-admin/MapPointInput.js

```diff
@@ -46,11 +58,14 @@ const MapPointInput = ({ classes, input }) => {
           <span className={classes.marker} />
         </Marker>
       )}
-    </Map>
+    </Map>))
   );
 };
 
 export default compose(
   addField,
   withStyles(styles),
+  connectAppProvider(({ env: { configMap } }) => ({
```

Comment:
What about creating a small `withMapConfig` HOC to abstract provider connection complexity?

Reply:
Agree, Done !